test/e2e/magellan.json

```diff
@@ -1,6 +1,6 @@
 {
 	"mocha_tests": [ "specs" ],
-	"mocha_args": "-R spec-junit-reporter",
+	"mocha_args": "--file lib/mocha-hooks/mocha-hooks.js --reporter mocha-teamcity-reporter -R spec-junit-reporter",
```

Comment:
Out of curiosity, why doesn't this work via `.mocharc.js`? Or is that used for something else?

Reply:
That's a good point. I think specifying `.mocharc.js` should work here. I will try it out.

Reply:
Good call, it works - thanks for pointing this out! 

This also means I should be able to remove my changes to `.teamcity/_self/projects/WebApp.kt` without breaking any of the CircleCI or TeamCity workflows.